src/main/scala/cromwell/engine/package.scala

```diff
@@ -30,6 +30,8 @@ import scala.util.{Failure, Success, Try}
  * Internally, this package is built on top of [[cromwell.binding]].
  */
 package object engine {
+  private val DEFAULT_CALL_CACHING_VALUE = false
```

Comment:
My old Java demons probably woke up when you said capitalized. Damn it I knew this "Java exorcist" on craigslist was a scam...